src/server/api/boards/boards.js

```diff
@@ -100,6 +100,47 @@ function addCardToColumn(board, columnId, cardName) {
   });
 }
 
+function validateColumnId(board, columnId, newColumnId) {
+  let oldFound = false;
+  let newFound = false;
+
+  board.columns.forEach(function(element) {
```

Comment:
`element` is a column so you can name it `column` instead. it gives you more meaning inside the function